php-8.3-mysql/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG ARCH=amd64
+FROM sykescottages/php:8.3-cli-$ARCH
+
+ENV TZ=Europe/London
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get -qq update && \
+    apt-get -qq install -y \
+    php8.3-mysql \
+    php8.3-mcrypt \
+    php8.3-gd \
+    php8.3-ldap \
+    php8.3-amqp \
+    mysql-client
+
+COPY --from=composer:2 /usr/bin/composer /usr/bin/composer
```

Comment:
Not needed as the base PHP 8 image does this https://github.com/SykesCottages/docker-php/blob/master/8-base/Dockerfile#L11